src/window.ts

```diff
@@ -83,48 +85,48 @@ export const openBlankChildWindow = ({
-  } catch (e) {}
-
-  if (newWindowDocument) {
-    newWindowDocument.write(
-      '<html><head><meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no"><title>Supertab</title></head>',
-    );
-    newWindowDocument.close();
-
-    loadOmegaAnimation()
-      .then((omegaAnimation) => {
-        if (
-          newWindowDocument &&
-          newWindowDocument.getElementById("supertab-loading-animation") ===
-            null
-        ) {
-          const container = newWindowDocument.createElement("div");
-          container.id = "supertab-loading-animation";
-          container.style.cssText =
-            "display: flex; flex-direction: column; align-items: center; justify-content: center; height: 100%";
-
-          const img = newWindowDocument.createElement("img");
-          img.src = omegaAnimation;
-          img.style.cssText = "width: 180px; height: auto";
-
-          const text = newWindowDocument.createElement("p");
-          text.style.cssText =
-            "margin-top: -10px; font-size: 16px; font-weight: 400; color: #555; font-family: Helvetica";
-          text.textContent = "Loading your Supertab...";
-
-          container.appendChild(img);
-          container.appendChild(text);
-          newWindowDocument.body.appendChild(container);
-        }
-      })
-      // eslint-disable-next-line
-      .catch((e) => {});
-  }
+  //let newWindowDocument = null;
+
+  //try {
+  //  newWindowDocument = newWindow?.document;
+  //  // eslint-disable-next-line
+  //} catch (e) {}
+
+  //if (newWindowDocument) {
+  //  newWindowDocument.write(
+  //    '<html><head><meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no"><title>Supertab</title></head>'
+  //  );
+  //  newWindowDocument.close();
+
+  //  loadOmegaAnimation()
+  //    .then((omegaAnimation) => {
+  //      if (
+  //        newWindowDocument &&
+  //        newWindowDocument.getElementById("supertab-loading-animation") ===
+  //          null
+  //      ) {
+  //        const container = newWindowDocument.createElement("div");
+  //        container.id = "supertab-loading-animation";
+  //        container.style.cssText =
+  //          "display: flex; flex-direction: column; align-items: center; justify-content: center; height: 100%";
+
+  //        const img = newWindowDocument.createElement("img");
+  //        img.src = omegaAnimation;
+  //        img.style.cssText = "width: 180px; height: auto";
+
+  //        const text = newWindowDocument.createElement("p");
+  //        text.style.cssText =
+  //          "margin-top: -10px; font-size: 16px; font-weight: 400; color: #555; font-family: Helvetica";
+  //        text.textContent = "Loading your Supertab...";
+
+  //        container.appendChild(img);
+  //        container.appendChild(text);
+  //        newWindowDocument.body.appendChild(container);
+  //      }
+  //    })
+  //    // eslint-disable-next-line
+  //    .catch((e) => {});
+  //}
```

Comment:
@tomasstark Can you help me understand why this is commented out?

Reply:
@jmcodingde Ah! My local working copy from some experimentation before propagated into this PR! 😮 Thank you for pointing it out. Removing…